api/src/main/java/org/apache/iceberg/PartitionSpec.java

```diff
@@ -109,9 +109,9 @@ public int specId() {
       PartitionField field = fields[i];
       Type sourceType = schema.findType(field.sourceId());
       Type resultType = field.transform().getResultType(sourceType);
-      // assign ids for partition fields starting at PARTITION_DATA_ID_START to leave room for data file's other fields
+      // assign ids for partition fields starting at 1000 to leave room for data file's other fields
```

Comment:
This is no longer assigning IDs, so the comment can be removed.

Reply:
let me take care.